sklearn/semi_supervised/label_propagation.py

```diff
@@ -358,6 +362,18 @@ class distributions will exceed 1 (normalization may be desired).
             affinity_matrix /= normalizer[:, np.newaxis]
         return affinity_matrix
 
+    def fit(self, X, y):
+        # alpha is deprecated for LabelPropagation because it doesn't have any
+        # theoretical meaning (from the reference paper). Look at PR 6727.
+        if self.alpha is not None:
+            warnings.warn(
+                "Deprecated in 0.19 and it's going to be removed in 0.21.",
+                DeprecationWarning
+            )
```

Comment:
Should we be setting _alpha to the user-provided alpha until deprecation is compeleted? Otherwise our deprecation warning should state that alpha is being ignored.